horizon/static/framework/widgets/modal/hz-submit-modal.controller.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * (c) Copyright 2016 Hewlett Packard Enterprise Development Company LP
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+(function() {
+  'use strict';
+
+  /**
+   * @ngdoc controller
+   * @name HzSubmitModalController
+   *
+   * @description
+   * Horizon's controller for submit modal dialog.
+   */
+  angular
+    .module('horizon.framework.widgets.modal')
+    .controller('horizon.framework.widgets.modal.HzSubmitModalController', HzSubmitModalController);
+
+  HzSubmitModalController.$inject = [
+  ];
+
+  function HzSubmitModalController() {
+    var ctrl = this;
+    return ctrl;
```

Comment:
I don't think you really return a control; it's the object itself so it doesn't need returning.